src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -490,7 +489,7 @@ const ManifestComponent = ({
             setManifestCodeEditorMode(ManifestCodeEditorMode.APPLY_CHANGES)
             setShowLockedDiffModal(true)
         } else {
-            handleCallApplyChangesAPI(modifiedManifestString).then(noop).catch(noop)
+            await handleCallApplyChangesAPI(modifiedManifestString)
```

Comment:
Since we moved to async await. We should wrap with try/catch as per then/catch logic

Reply:
the above promise only resolves never rejects so we can ignore here